core/src/components/datetime-button/test/a11y/datetime-button.e2e.ts

```diff
@@ -41,10 +41,10 @@ configs({ directions: ['ltr'], modes: ['ios'] }).forEach(({ title, screenshot, c
 
       const datetimeButton = page.locator('ion-datetime-button');
 
-      await expect(datetimeButton).toHaveScreenshot(screenshot(`datetime-button-scale-truncate`));
+      await expect(datetimeButton).toHaveScreenshot(screenshot(`datetime-button-scale-wrap`));
     });
 
-    test('should truncate text on larger font sizes inside of an item', async ({ page }) => {
+    test('should wrap text on larger font sizes inside of an item', async ({ page }) => {
```

Comment:
Optional: We might be able to remove this test if the behavior doesn't need any special CSS rules to work. I added it because the item specifically needed that `min-width: 0` added in order to truncate.

Reply:
Oh ok good to know! I'll remove it then

Reply:
Removed in https://github.com/ionic-team/ionic-framework/pull/27872/commits/bac8632bc70dc424c4b08e875efe3f692a92f7d3